### PR TITLE
Restructure cmd package for maintainability

### LINKED ISSUE #300: Improve cmd package test coverage
This package ins untested and suffer from bad practices.

It needs to be cleaned up and tested

### PR DESCRIPTION
**Pull request by ValentinTrinque**
Close #300

### CODE REVIEW

**Review by jeremyletang — APPROVED** [review]

File: cmd/key_generate.go [hunk 1]
```diff
@@ -2,183 +2,175 @@ package cmd
...
 		if err != nil {
-			return fmt.Errorf("couldn't create wallet: %w", err)
+			return nil, fmt.Errorf("couldn't initialise wallets store: %w", err)
 		}
 
-		if rootArgs.output == "human" {
-			p.CheckMark().Text("Wallet ").Bold(keyGenerateArgs.wallet).Text(" has been created at: ").SuccessText(store.GetWalletPath(keyGenerateArgs.wallet)).NextLine()
+		// Because the passphrase needs to be retrieved based on the state of the
+		// specified wallet, we get it here and fill the request with it.
+		passphrase, err := passphraseGetter(s.WalletExists(req.Wallet))
```
Comment (wwestgarth) [inline cmd/key_generate.go:46]: Late to the party: Was just having a play around, and I think this should be `!s.WalletExists(req.Wallet)`? We want the confirmation when the wallet is created and not every subsequent time we use the wallet.

Comment (ValentinTrinque) [reply cmd/key_generate.go:46]: 🦅 👁️

File: cmd/key_annotate.go [hunk 2]
```diff
@@ -2,88 +2,166 @@ package cmd
...
-		} else {
-			p.CheckMark().SuccessText("Annotation succeeded").NextSection()
-			p.Text("New metadata:").NextLine()
-			printMeta(p, metadata)
-		}
+	return req, nil
+}
+
+func PrintAnnotateKeyResponse(w io.Writer, req *wallet.AnnotateKeyRequest) {
+	metadataHaveBeenCleared := len(req.Metadata) != 0
```
Comment (wwestgarth) [inline cmd/key_annotate.go:157]: I think this should be `metadataHaveBeenCleared := len(req.Metadata) == 0`? Locally I'm seeing it print "Annotation cleared." when setting new metadata.

Comment (ValentinTrinque) [reply cmd/key_annotate.go:157]: Nice catch !

**Pull request merged**